rotkehlchen/data_import/importers/binance.py

```diff
@@ -63,7 +63,7 @@ class BinanceSingleEntry(BinanceEntry, metaclass=abc.ABCMeta):
     """
     AVAILABLE_OPERATIONS: tuple[str, ...]
 
-    def is_entry(self, requested_operation: str) -> bool:
+    def is_entry(self, requested_operation: str, account: str, change: int) -> bool:  # pylint: disable=unused-argument
```

Comment:
you need to adjust the docstring to explain what `account` and `change` are used for.